charts/wildfly/values.schema.json

```diff
@@ -659,6 +691,15 @@
                   }
                 }
             }
+        },
+        "extraObjects": {
```

Comment:
That's a good idea.
Another approach would be to create a Helm chart that _depends_ on the `wildfly` chart as a lib and add others resources. This would be better for "distinct" resources (eg a database deployment).
But for your use case, the PVC is intrinsically tied to the WildFly deployment and it makes sense to deploy it as an atomic release.
Another good use case would be config maps or secrets